index.js

```diff
@@ -566,8 +566,7 @@ alexa.app = function(name) {
     }
 
     if (options.expressApp && options.router) {
-      console.warn("Since alexa-app@4.0.0, you are no longer required to pass both 'expressApp' and 'router'.");
-      console.warn("When both passed, 'alexa-app' would be attached to 'expressApp' instance.");
+      console.warn("Both 'expressApp' and 'router' are specified, attaching to 'expressApp' only.\nMore details on https://github.com/alexa-js/alexa-app/blob/master/UPGRADING.md");
```

Comment:
I think it should either be an exception or be backward compatible and produce a warning that this behavior is deprecated. 

If we ignore `router`, the user is not getting the behavior from the previous release and we just tell them "sorry, it's not going to work, but we are going to pretend like it does" :)

I actually would prefer a backward compatible version here with a warning, because that allows a lot more flexibility in combining versions of alexa-app-server and alexa-app.

What do you think?